### PR TITLE
Feature/multiple bible translations

### PR DESCRIPTION
**Pull request by fortune710**
## Description
Created a feature to enable bible verse results for multiple translations to improve accessibility.

## Related Issues
This PR resolves issue #7

## Changes List
- [x] Created a centralized location for API implementation
- [x] Enabled API response caching for better performance
- [x] Refactored some code for better readability

## Type of Changes

- [x] Enhancement (improves or changes existing functionality)

## Checklist
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Namit2111** [comment]
please add screenshot of output in PR

File: backend/requirements.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Jinja2==3.1.4
 joblib==1.4.2
 MarkupSafe==2.1.5
 nltk==3.9.1
-numpy==2.1.0
+numpy==2.0.2
```
Comment (Namit2111) [inline backend/requirements.txt]: remove this we need 2.1.0

Comment (fortune710) [reply backend/requirements.txt]: okay

**Comment by fortune710** [comment]
Done with corrections
Here's the output:

Screenshot 2024-10-18 194939

**Comment by Namit2111** [comment]
Hi @fortune710 please resolve the conflicts that are there , also add more example images.
Create a new PR with these changes also remember to squash your commits before making a PR

**Comment by JustinhSE** [comment]
@fortune710 thanks for the screenshot, this looks great! But do try building the project in your local repo and see what issues might be causing this to fail

**Comment by fortune710** [comment]
Hey @JustinhSE. I did run the build locally, perhaps it was the conflicts from the previous merge, but I have resolved that and the build works as it should. Here the results of the local build ⬇
next-build

I'll make another PR for this

**Comment by JustinhSE** [comment]
@Namit2111

**Comment by Namit2111** [comment]
There are conflicts in this branch resolve those after that create a PR with Squashed commits

**Comment by JustinhSE** [comment]
@fortune710 resolve my comments and the merge conflicts as well first, then make another PR. Another pr was merged earlier today so you may need to pull the changes.

**Comment by fortune710** [comment]
@JustinhSE  I resolved the merge conflicts some moments ago from the landing page and I pushed the changes to my fork, the build result I sent earlier was from after the merge conflict resolution.

**Comment by JustinhSE** [comment]
Okay great! Yea open a new pr and we will review it

**Pull request closed without merging**